app/views/posts/show.html.erb

```diff
@@ -19,20 +19,25 @@
 
 <%= render 'posts/expanded', post: @post, float_notice: false %>
 
+<% num_answers = @children.count %>
```

Comment:
Info: does this count all answers or only answers the viewing user can see?  (The PR shows an example for people who can see deleted answers; just want to make sure we're doing the right thing for people who can't.)

Reply:
Specifically only those that the user can see, yes - `@children` is a local variable set by the controller that does the check (the same one is currently used to display the list of answers)